frontend/src/components/layout/header.tsx

```diff
@@ -39,6 +40,7 @@ const Header = () => {
               </NavItem>
             </Nav>
             <div className="nav-item item-center">
+              <DarkModeSwitch />
```

Comment:
I think we should move dark mode switcher to the footer, like the translation component